src/System.Resources.ResourceManager/tests/System.Resources.ResourceManager.Tests.csproj

```diff
@@ -34,9 +34,14 @@
       <Generator>ResXFileCodeGenerator</Generator>
       <LastGenOutput>TestResx.Designer.cs</LastGenOutput>
     </EmbeddedResource>
+    <EmbeddedResource Include="Resources\TestResx.es.resx"/>
+    <EmbeddedResource Include="Resources\TestResx.es-MX.resx"/>
+    <EmbeddedResource Include="Resources\TestResx.fr-FR.resx"/>
+    <EmbeddedResource Include="Resources\TestClassWithoutNeutralResources.fr.resx"/>
     <SupplementalTestData Include="Resources\TestResx.netstandard17.resources" />
     <SupplementalTestData Include="Resources\bitmap.bmp" />
     <SupplementalTestData Include="Resources\icon.ico" />
+    <SupplementalTestData Include="$(OutputPath)\**\System.Resources.ResourceManager.Tests.resources.dll" />
```

Comment:
While this seemed to work when I tested locally on Ubuntu 16.04 build.  It is not working in the CI builds.  Maybe it is missing a dependency?

Reply:
Maybe @ViktorHofer knows the right answer.

Reply:
When I do a clean build locally I am seeing the same failure result as in CI.  This doesn't work because the satellite assemblies are not built before the copy occurs.

In reality these files are not `<SupplementalTestData>` but rather satellite assemblies of the test assembly.  They should be copied as part of copying the build products.

Reply:
An easy/ugly workaround is to copy these assemblies in a copy task after the build with the `$(TestPath)` as the DestinationFolder. The right fix would be to adjust the `BinPlaceItem` collection.

cc @ericstj

Reply:
There is a defect open for this dotnet/buildtools/issues/246

Reply:
OK I added the workaround

```xml
  <!--
    Workaround lack of BuildTools support for copying satellite assemblies
    https://github.com/dotnet/buildtools/issues/246
  -->
  <PropertyGroup>
    <TestDependsOn>
      $(TestDependsOn);
      CopyMissingSatelliteAssemblies
    </TestDependsOn>
  </PropertyGroup>
  <Target Name="CopyMissingSatelliteAssemblies" AfterTargets="BinPlace" BeforeTargets="RunTestsForProject">
    <Copy SourceFiles="$(OutputPath)/fr-FR/System.Resources.ResourceManager.Tests.resources.dll"
          DestinationFolder="$(TestPath)/fr-FR">
    </Copy>
    <Copy SourceFiles="$(OutputPath)/fr/System.Resources.ResourceManager.Tests.resources.dll"
          DestinationFolder="$(TestPath)/fr">
    </Copy>
    <Copy SourceFiles="$(OutputPath)/es-MX/System.Resources.ResourceManager.Tests.resources.dll"
          DestinationFolder="$(TestPath)/es-MX">
    </Copy>
    <Copy SourceFiles="$(OutputPath)/es/System.Resources.ResourceManager.Tests.resources.dll"
          DestinationFolder="$(TestPath)/es">
    </Copy>
  </Target>
```

Reply:
If you do that please add Inputs and Outputs attributes to the Target element, containing the files. I think it's cleanest something like 
```xml
<ItemGroup>
     <SatelliteAssemblyToCopy Include="$(OutputPath)/fr-FR/System.Resources.ResourceManager.Tests.resources.dll" DestinationFolder="$(TestPath)/fr-FR"/>
     ...
</ItemGroup>

  <Target Name="CopyMissingSatelliteAssemblies" AfterTargets="BinPlace" BeforeTargets="RunTestsForProject"
          Inputs="@(SatelliteAssemblyToCopy)" Outputs="%(SatelliteAssemblyToCopy.DestinationFolder)">
    <Copy SourceFiles="@(SatelliteAssemblyToCopy)" DestinationFolder="%(SatelliteAssemblyToCopy.DestinationFolder)"/>
  </Target>
```

Then run the build twice and confirm that no copying happened the second build.

Reply:
@danmosemsft I originally had it something like that, but when I added the `DestinationFolder` attribute the docs implied there was only one folder per copy command (not per file).  https://docs.microsoft.com/en-us/visualstudio/msbuild/copy-task?view=vs-2017

Reply:
That's true, but the Metadata expression %(...) causes MSBuild to run the task one for each distinct value. @viktorhofer can give pointers if necessary

Reply:
If that's true then this should work too.  I would rather not maintain an explicit list of satellite assemblies
```xml
  <!--
    Workaround lack of BuildTools support for copying satellite assemblies
    https://github.com/dotnet/buildtools/issues/246
  -->
  <PropertyGroup>
    <TestDependsOn>
      $(TestDependsOn);
      CopyMissingSatelliteAssemblies
    </TestDependsOn>
  </PropertyGroup>
  <Target Name="CopyMissingSatelliteAssemblies" AfterTargets="BinPlace" BeforeTargets="RunTestsForProject">
    <Copy SourceFiles="$(OutputPath)/**/*.resources.dll"
          DestinationFolder="$(TestPath)/%(RecursiveDir)">
    </Copy>
  </Target>
```

Reply:
@danmosemsft The edited version of the above comment is working.  
The recursive wild card ItemGroup SatelliteAssemblies can only be evaluated after BinPlace (Target is built).  So it made sense to me to leave it inside the `Target`.  Are you OK with this version? I'll force push it shortly...